test/server/test-run-commands-test.js

```diff
@@ -1581,5 +1581,142 @@ describe('Test run commands', function () {
                 }
             );
         });
+
+        it('Should validate UploadFileСommand', function () {
```

Comment:
Add tests: `Should create UploadCommand from object` and `Should create ClearUploadCommand from object`